test/web_console/context_test.rb

```diff
@@ -0,0 +1,28 @@
+require 'test_helper'
+
+module WebConsole
+  class ContextTest < ActiveSupport::TestCase
+    test '#extract(empty) includes local variables' do
```

Comment:
If this can be empty so often, can we actually default it to something blank, say `nil`.